airflow/bin/cli.py

```diff
@@ -1433,8 +1435,9 @@ def users_manage_role(args, remove=False):
             args.username or args.email))
 
     role = appbuilder.sm.find_role(args.role)
+    valid_roles = appbuilder.sm.get_all_roles()
```

Comment:
Can you move this inside of the `if` statement?

Reply:
Done